test/e2e/common/node/container_probe.go

```diff
@@ -222,6 +222,49 @@ var _ = SIGDescribe("Probing container", func() {
 		RunLivenessTest(ctx, f, pod, 0, defaultObservationTimeout)
 	})
 
+	/*
```

Comment:
I validated the diff of moved code is only `ginkgo.It` -> `framework.ConformanceIt`. Can you please move this code back into original place so the git blame will be easier to run in future.

Reply:
OK, done